retroshare-android-service/src/service.cpp

```diff
@@ -39,11 +39,19 @@ int main(int argc, char *argv[])
 #endif
 
 	QCoreApplication a(argc, argv);
+
+	RsInit::InitRsConfig();
```

Comment:
is it a linker error?
If yes, then libretroshare has to come after libresapi on the linker command line.